channels/es.m3u

```diff
@@ -1,11 +1,11 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidespain.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas HD
 http://hlsliveamdgl8-lh.akamaihd.net/i/hlsdvrlive_1@583030/index_1500_av-p.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas
 http://hlsliveamdgl8-lh.akamaihd.net/i/hlsdvrlive_1@583030/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/sADbx7S.png" group-title="",25 TV
 http://cdnlive.shooowit.net/25televisiolive/25tv.smil/.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/M0JqHa9.png" group-title="Local",7 Murcia
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/M0JqHa9.png" group-title="Local",7 Murcia HD
```

Comment:
Same here:
<img width="393" alt="Captura de pantalla 2020-04-29 a las 14 58 01" src="https://user-images.githubusercontent.com/7253922/80593441-ecd69700-8a29-11ea-8398-c5a146e525b0.png">

Reply:
They are supposed to be HD channels, if they don't broadcast in HD, what are we going to do ...
Equally, if they are not HD.
Why repeat the channel in the list with the same name?
In such a case you would simply delete the repeated line.

Reply:
I think you're right, the second line should be deleted.